test/framework/src/main/java/org/elasticsearch/test/AbstractQueryTestCase.java

```diff
@@ -737,7 +737,7 @@ public void testToString() throws IOException {
         for (int runs = 0; runs < NUMBER_OF_TESTQUERIES; runs++) {
             QB testQuery = createTestQueryBuilder();
             XContentType xContentType = XContentType.JSON;
-            String toString = testQuery.toString();
+            String toString = Strings.toString(testQuery);
```

Comment:
this way you don't really test the toString :) I think it's good to leave testQuery.toString(). We will later change the toString to not depend on ToXContentToBytes.